src/components/contact/ContactForm.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from "react";
+import { FaFacebookF, FaTwitter, FaLinkedinIn } from "react-icons/fa";
+import { AiFillInstagram } from "react-icons/ai";
+
+const ContactForm = () => {
+  const [firstName, setFirstName] = useState<string>();
```

Comment:
let ts infer and dont leave initial state undefined

```ts
const [message, setMessage] = useState("");
```

Reply:
Done